packages/fluentui/react-northstar/src/components/Chat/ChatMessageContent.tsx

```diff
@@ -0,0 +1,65 @@
+import { compose } from '@fluentui/react-bindings';
+import * as PropTypes from 'prop-types';
+
+import { commonPropTypes } from '../../utils';
+import { Box, BoxProps, BoxStylesProps } from '../Box/Box';
+import { ChatDensity } from './chatDensity';
+import { ChatMessageLayout } from './ChatMessage';
+
+export interface ChatMessageContentOwnProps {
```

Comment:
these props are simply a copy + paste from ChatMessage right ? could you just do something like

```ts
interface ChageMessageContentOwnProps  extends Pick<ChatMessageProps, 'badgePosition' | 'density'> {}
```

Reply:
Right and it's a good idea. Updated.